qt/scientific_interfaces/Inelastic/BayesFitting/Quasi.cpp

```diff
@@ -518,13 +528,16 @@ void Quasi::plotClicked() {
   auto const resultWS = AnalysisDataService::Instance().retrieveWS<MatrixWorkspace>(resultName);
   int const numSpectra = (int)resultWS->getNumberHistograms();
   IndirectTab::checkADSForPlotSaveWorkspace(resultName, true);
-  auto const paramNames = {"Amplitude", "FWHM", "Beta"};
+  auto const paramNames = {"amplitude", "fwhm", "beta"};
   for (std::string const &paramName : paramNames) {
 
-    if (plot == paramName || plot == "All") {
+    if (plot == paramName || plot == "all") {
```

Comment:
I'm happy to gatekeeper this, but in case it's helpful for the future I just thought I'd mention that there seem to be a few string literals in use here and it may help with maintainability to consider defining some as constants at some point.